com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumeAsset.cs

```diff
@@ -126,11 +126,13 @@ internal bool ResolveSharedCellData(TextAsset cellSharedDataAsset, TextAsset cel
             return true;
         }
 
-        internal bool ResolvePerStateCellData(TextAsset cellDataAsset, TextAsset cellOptionalDataAsset)
+        internal bool ResolvePerStateCellData(int index, TextAsset cellDataAsset, TextAsset cellOptionalDataAsset)
         {
             if (cellDataAsset == null)
                 return false;
 
+            bool hasTwoStates = (index == 1);
```

Comment:
Comment here pls

Reply:
i changed this logic because it didn't work when switching to a non baked state